app/(service)/recruit-detail/[shopId]/[recruitId]/_components/recruitCard.tsx

```diff
@@ -0,0 +1,104 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+
+import Image from 'next/image';
+import { useParams } from 'next/navigation';
+
+import Button from '@/components/button/button';
+import PercentageBadge from '@/components/post/percentageBadge';
+import apiClient from '@/libs/axios';
+import calculatePercentage from '@/utils/calculatePercentage';
+import formatDateRange from '@/utils/formatDateRange';
+
+interface FetchData {
+  hourlyPay: 'number';
+  startsAt: 'string';
+  workhour: 'number';
+  description: 'string';
+  closed: 'boolean';
+  shop: {
+    item: {
+      name: 'string';
+      category: 'string';
+      address1: 'string';
+      description: 'string';
+      imageUrl: 'string';
+      originalHourlyPay: 'number';
+    };
+  };
+}
+
+function RecruitCard() {
+  const [recruitData, setRecruitData] = useState<FetchData | null>(null);
+  const { shopId, recruitId } = useParams();
+  // 임시로 쓴 데이터 요청임
+  useEffect(() => {
```

Comment:
P4: 아래와 같이 한 줄 띄워서 코드를 분리시켜 주시면 가독성이 더 좋을 것 같습니다 :)

```suggestion
  const { shopId, recruitId } = useParams();
  
  // 임시로 쓴 데이터 요청임
  useEffect(() => {
```

Reply:
확인했습니다